### DISCUSSION TITLE
:class incorrectly applied when using wire:navigate and browser back button

### DISCUSSION THREAD

**Discussion by bezanis** [alpinejs/alpine#3749] [7. Bugs (archived)]
While using Livewire 3.0.1 with the included version of Alpine along with Tailwind, I'm experiencing an issue with a class being applied after wire:navigate followed by a browser back button press.
To replicate:
Add a div with a x-data param. 
Inside that, another div has a :class with a text class added conditionally based on the value in the x-data. If I navigate to another page normally, the :class logic gets reapplied correctly when I use the browser back to return to the page. 
However, if I use wire:navigate to leave the page, and then use the browser back button, the conditional logic is incorrectly still applied to the previous state, along with the default state.

Code:
```
<div  x-data="{ locations:'500' }">
    <div class="text-gray">
        <div @click="locations='1'" :class="(locations === '1') && 'text-orange'">1 Location</div>
        <div @click="locations='500'" :class="(locations === '500') && 'text-orange'">500 Locations</div>
    </div>
</div>
```
In this situation, locations is initialized to 500. The 1Location text starts as gray and the 500 Locations text starts as orange. 
I click '1 Location' which turns 1 Location orange and 500 Locations gray. If I then use a link on the page with wire:navigate to leave the page, and then use the browser back button, both 1 Location and 500 Locations texts are orange. However, if I instead navigate away from the page using a normal link, then return, the colors are reset correctly.

**Comment by joshhanley** [comment]
@bezanis please open this on the Livewire repo as I believe it will be a problem with `wire:navigate` not Alpine 🙂

**Reply by bezanis** [reply]
Opened in livewire repo https://github.com/livewire/livewire/discussions/6686
